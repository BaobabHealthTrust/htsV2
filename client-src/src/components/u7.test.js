import React from 'react';
import ReactDOM from 'react-dom';
import U7 from './u7';
import renderer from 'react-test-renderer';
import toJson from 'enzyme-to-json';
import Enzyme, {
    shallow
} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({
    adapter: new Adapter()
});

describe('Component: U7', () => {

    it('should match its empty snapshot', () => {
        const tree = renderer.create( 
            <U7 />
        ).toJSON();

        expect(tree).toMatchSnapshot();
    })

})