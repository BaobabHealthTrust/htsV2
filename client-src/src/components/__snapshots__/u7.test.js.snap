// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: U7 should match its empty snapshot 1`] = `
<div
  className="headTab"
  id="tableDiv2"
>
  <table
    cellPadding="1"
    style={
      Object {
        "fontSize": "12px",
      }
    }
    width="100%"
  >
    <tbody>
      <tr>
        <td
          className="blueText"
          style={
            Object {
              "fontWeight": "bold",
              "textAlign": "right",
              "width": "70px",
            }
          }
        >
          Gender
        </td>
        <td
          style={
            Object {
              "textAlign": "center",
              "width": "3px",
            }
          }
        >
          :
        </td>
        <td
          align="left"
          id="gender"
        />
      </tr>
      <tr>
        <td
          className="blueText"
          style={
            Object {
              "fontWeight": "bold",
              "textAlign": "right",
            }
          }
        >
          Age
        </td>
        <td
          style={
            Object {
              "textAlign": "center",
              "width": "3px",
            }
          }
        >
          :
        </td>
        <td
          align="left"
          id="other_id"
        />
      </tr>
      <tr>
        <td
          className="blueText"
          style={
            Object {
              "fontWeight": "bold",
              "textAlign": "right",
            }
          }
        >
          Address
        </td>
        <td
          style={
            Object {
              "textAlign": "center",
              "width": "3px",
            }
          }
        >
          :
        </td>
        <td
          align="left"
          id="addressl1"
        />
      </tr>
      <tr>
        <td
          className="blueText"
          style={
            Object {
              "fontWeight": "bold",
              "textAlign": "right",
            }
          }
        >
           
        </td>
        <td
          style={
            Object {
              "textAlign": "center",
              "width": "3px",
            }
          }
        >
           
        </td>
        <td
          align="left"
          id="addressl2"
        />
      </tr>
      <tr>
        <td
          className="blueText"
          style={
            Object {
              "fontWeight": "bold",
              "textAlign": "right",
            }
          }
        >
           
        </td>
        <td
          style={
            Object {
              "textAlign": "center",
              "width": "3px",
            }
          }
        >
           
        </td>
        <td
          align="left"
          id="addressl3"
        />
      </tr>
      <tr>
        <td
          className="blueText"
          style={
            Object {
              "fontWeight": "bold",
              "textAlign": "right",
            }
          }
        >
          Phone
        </td>
        <td
          style={
            Object {
              "textAlign": "center",
              "width": "3px",
            }
          }
        >
          :
        </td>
        <td
          align="left"
          id="phone"
        />
      </tr>
    </tbody>
  </table>
</div>
`;
